room.js

```diff
@@ -1,703 +0,0 @@
-                break
-
-            case 'fine' && this.health === 100:
-                break
-
-            default:
-                console.log("Error, unusual status detected!")
-                break
-        }
-    },
-    
-    game_over(){
-        if(this.health <= 0){
-            console.log('\n \x1b[33m%s\x1b[0m', "Health has reached 0. Game over!")
-            process.exit()
-        }
-    }
-    
-}
-
-
-// async function player_action(action){
-    
-// }
-/// CLASSES /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-class box {
-    constructor(name, locked, boarded, key, loot, desc){
-        this.name
-        this.locked = locked
-        this.boarded = boarded
-        this.key = key
-        this.desc = desc
-        this.loot = loot
-    }
-    use_crowbar(){
-        if(this.boarded === true &&  this.locked === false){
-            this.boarded = false
-            this.desc = `The ${this.name} no longer has any planks on it. Looks like whatever is inside is free for the taking?`
-            console.log('\n \x1b[33m%s\x1b[0m', 'You pull the boards off the box using the crowbar')   
-        } else if (this.boarded === true && this.locked === true){
-            console.log('\n \x1b[33m%s\x1b[0m', 'You can\'t pull off the boards until the metal lock in the way gets removed!') 
-        } else {
-            console.log('\n \x1b[33m%s\x1b[0m', 'There\'s no boards on here!')            
-        }
-    }
-
-    unlock(){
-        if(this.locked === true && player.inv.includes(this.key)){ // Note for me: Refactor this code later to call for the key's object name in the player's inventory. For now it just checks for a string.
-            this.locked = false
-            this.desc = (`\n It\'s a ${this.name} boarded up now without that pesky metal lock in the way!`)
-            room1.south_description = "The box has forgone its relationship with his pal the metal lock."
-            console.log('\n \x1b[33m%s\x1b[0m', 'The metal lock falls to the ground upon being unlocked with a loud THUD')
-        } else if(this.locked === false){
-            console.log("\n It's already unlocked!")
-        } else {
-            console.log("You need the right key!")
-        }
-
-    }
-
-    open(){
-        if(this.locked === false && this.boarded === false){
-            console.log(`\n You open the ${this.name} to get a ${this.loot}!`)
-            player.inv.push(this.loot)
-        }
-    }
-
-
-} 
-
-/// OBJECTS /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-let room1_drawer_test_object = new box('drawer', false, false, 'none', 'drawer key', 'A wooden drawer slightly open with a note written on top!')
```

Comment:
I don't believe that this class is actually being used anywhere. You create "room1_Drawer_test_object" but after that, that object is never referenced again. If that's the case you could remove this code to make your project more condensed.